UnityProject/Assets/Scripts/Messages/Client/VariableViewer/OpenBookIDNetMessage.cs

```diff
@@ -16,7 +16,7 @@ public override void Process(NetMessage msg)
 
 		private void ValidateAdmin(NetMessage msg)
 		{
-			if (IsFromAdmin() == false) return;
+			if (IsFromAdmin("VV") == false) return;
```

Comment:
w?

Reply:
variable viewer VV

Reply:
`VARIABLE_VIEWER` then, no reason to be a cheapskate with letters

Reply:
cool